cypress/integration/ColorPicker.spec.ts

```diff
@@ -194,32 +194,38 @@ describe('ColorPicker', () => {
       });
 
       context('when a swatch is clicked', () => {
-        const color = '8660d1';
+        const color = '#ff5347';
+
         beforeEach(() => {
-          cy.get(`.wdc-color-picker--color-${color}`).click();
+          getSwatch(color).click();
         });
 
         it('should update the color input to the swatch color', () => {
-          getColorInput().should('have.value', color.toUpperCase());
+          getColorInput().should('have.value', color.slice(1).toUpperCase());
         });
 
         it('should have check icon', () => {
-          getColorInput().click();
-          cy.get(`.wdc-color-picker--color-${color}`)
+          getColorInput().focus();
+          getSwatch(color)
             .find('.wd-icon')
             .should('exist');
         });
-      });
 
-      context('when color reset is clicked', () => {
-        beforeEach(() => {
-          cy.get(`.wdc-color-picker--color-8660d1`).click();
-          getColorInput().click();
-          cy.get('[data-testid="color-picker-reset"]').click();
-        });
+        context('when color reset is clicked', () => {
```

Comment:
Nested this one a level deep, we shouldn't have to click on a swatch again when we can just put it under the `when swatch is clicked` context